community/server/src/main/java/org/neo4j/server/configuration/ConfigurationBuilder.java

```diff
@@ -137,4 +141,13 @@ public Configuration configuration()
         }
 
     }
+
+    class DefaultSettingsClassesHolder
```

Comment:
Why not just put the static method in the ConfigurationBuilder? It's already in that class, just wrapped in this inner class.